.coveragerc

```diff
@@ -1,7 +1,32 @@
 [run]
 source = EXOSIMS
+parallel=True
 omit = 
   EXOSIMS/e2eTests.py
   EXOSIMS/e2eTKTests.py
   EXOSIMS/util/KeplerSTM_C/CyKeplerSTM_setup.py
   EXOSIMS/SurveySimulation/tieredScheduler.py
+  EXOSIMS/util/plotConvergencevsNumberofRuns.py
+  EXOSIMS/util/plotKeepoutMap.py
+  EXOSIMS/util/plotPlanetPopRvsAandDetectedRvsA.py
+  EXOSIMS/util/plotTimeline.py
+  EXOSIMS/util/read_ipcluster_ensemble.py
+  EXOSIMS/util/RejectionSampler.py
+  EXOSIMS/util/runPostProcessing.py
+  EXOSIMS/util/yieldPlotHistograms.py
+  EXOSIMS/util/analysis_tools.py
+  EXOSIMS/util/collateAllUniqueDetections.py
+  EXOSIMS/util/csv_fix.py
+  EXOSIMS/util/depthOfSearch.py
+  EXOSIMS/util/evenlyDistributePointsOnSphere.py
+  EXOSIMS/util/fakeMultiRunAnalysis.py
+  EXOSIMS/util/fakeSingleRunAnalysis.py
+  EXOSIMS/util/getExoplanetArchive.py
+  EXOSIMS/util/kopparapuPlot.py
+  EXOSIMS/util/makeSimilarScripts.py
+  EXOSIMS/util/partitionSphere.py
+  EXOSIMS/util/planet_star_separation.py
+  EXOSIMS/util/plotC0vsT0andCvsT.py
+  EXOSIMS/util/plotCompletenessJointPDFs.py
+  EXOSIMS/util/memoize.py
+  EXOSIMS/util/vprint.py
```

Comment:
I don't think vprint should be omitted

Reply:
I'll add it back in with the next push. I wasn't quite sure how to test a console output, but I have some ideas now.